pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -745,6 +845,8 @@ def plot_3D_cell_morphology(
           is used)
 
     :type highlight_spec: dict
+    :param isPCA: bool that if True, transforms single Cell according to its PCA to make it look 'upwards'
+    :type isPCA: bool
```

Comment:
probably worth adding the `axes_pos` argument here too, so that users can activate it when viewing single cells too (same for schematic figure)?

Reply:
axes_pos argument doesn't exist in plot_interactive_3d either, should I add in every one of those?

Reply:
Ah, yeh. We can think about whether we want to also expose this in the `pynml-plotmorph` command line tool later.